quick-start.md

```diff
@@ -26,4 +26,4 @@ To install a simple, developer-friendly version of [all the services on one devi
 📚 How to [get support for Open Horizon](common-requests/get-technical-support.md)
 
 ### Learn More
-
+About Open Horizon Project [here](https://www.youtube.com/playlist?list=PLgohd895XSUddtseFy4HxCqTqqlYfW8Ix).
```

Comment:
Instead of linking the word "here", can you make the linked text more semantically meaningful?  Something descriptive like "video playlist" would let people know what to expect from clicking on the link (and it is better for SEO as well).

Reply:
Yes sure ..I can rectify that ..thanks for the review .:+1: